---
string[] dome_markers
string[] mild_markers